autoemulate/simulations/reactiondiffusion.py

```diff
@@ -0,0 +1,129 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from numpy.fft import fft2
+from numpy.fft import ifft2
+from scipy.integrate import solve_ivp
+from tqdm import tqdm
+
+integrator_keywords = {}
+integrator_keywords["rtol"] = 1e-12
+integrator_keywords["method"] = "RK45"
+integrator_keywords["atol"] = 1e-12
+
+
+def reaction_diffusion(t, uvt, K22, d1, d2, beta, n, N):
+    """
+    Define the reaction-diffusion PDE in the Fourier (kx, ky) space
+    """
+
+    ut = np.reshape(uvt[:N], (n, n))
+    vt = np.reshape(uvt[N : 2 * N], (n, n))
+    u = np.real(ifft2(ut))
+    v = np.real(ifft2(vt))
+    u3 = u**3
+    v3 = v**3
+    u2v = (u**2) * v
+    uv2 = u * (v**2)
+    utrhs = np.reshape((fft2(u - u3 - uv2 + beta * u2v + beta * v3)), (N, 1))
+    vtrhs = np.reshape((fft2(v - u2v - v3 - beta * u3 - beta * uv2)), (N, 1))
+    uvt_reshaped = np.reshape(uvt, (len(uvt), 1))
+    uvt_updated = np.squeeze(
+        np.vstack(
+            (-d1 * K22 * uvt_reshaped[:N] + utrhs, -d2 * K22 * uvt_reshaped[N:] + vtrhs)
+        )
+    )
+    return uvt_updated
+
+
+def simulate_reactiondiffusion(x, return_timeseries=False, n=32, L=20, T=10.0, dt=0.1):
```

Comment:
Very minor suggestion, perhaps this would be clearer like the fn above `def reaction_diffusion`:
```suggestion
def simulate_reaction_diffusion(x, return_timeseries=False, n=32, L=20, T=10.0, dt=0.1):
```